src/transports/ssh.c

```diff
@@ -934,7 +934,7 @@ int git_transport_ssh_global_init(void)
 		return -1;
 	}
 
-	git__on_shutdown(shutdown_ssh);
+	git_runtime_shutdown_register(shutdown_ssh);
```

Comment:
We're ignoring the function's return code here

Reply:
Fixed.